.NET/Microsoft.Recognizers.Text.DateTime/Extractors/BaseMergedDateTimeExtractor.cs

```diff
@@ -293,7 +311,7 @@ private List<ExtractResult> AddMod(List<ExtractResult> ers, string text)
 
                 if (!success)
                 {
-                    success = TryMergeModifierToken(er, config.SinceRegex, text);
+                    success = TryMergeModifierToken(er, config.SinceRegex, text, true);
```

Comment:
This is the only place where true is passed?
Can you add a code comment here explaining why? Also, please make it a named parameter in the call to TryMergeModifierToken.